src/index.html

```diff
@@ -57,7 +57,9 @@ <h3 class="recipe--title" maxlength="40">Recipe Name</h3>
               <p>Click for Instructions</p>
             </div>
           </div>
-          <h4 class="recipe--tags">tag information</h4>
+          <div class="recipe--bottom">
```

Comment:
Do we need this div anymore since we figured out how to use flex on the recipe--tags?